speech-commands/training/browser-fft/tflite_conversion.ipynb

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
<p>Is this the older model format? will it be better to upgrade the model with the latest version of tfjs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfjs-models/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>This is a layers model. So it's already the latest format because there is no newer version of the format AFAIK. </p><p> </p><p>However, the model json is indeed in an old format, in that the "config" field of the "sequential" model is an array instead of a dict as in the latest format. But both tensorflow and tensorflowjs ought to be backward compatible to this old format. Also, users may get this old format from Teachable Machine, so we should make sure we can handle it correctly.</p><p> </p>

Reply:
Thanks for the review, @pyu10055 

PTAL.